Gemfile

```diff
@@ -11,18 +11,19 @@ gem "elasticsearch-rails"
 gem "faraday"
 gem "faraday_middleware"
 gem "govuk_elements_rails"
-gem "govuk_publishing_components"
+gem "govuk_publishing_components", "~> 28.7.1" # Earliest release with Ruby 3+ support
 gem "htmlentities"
 gem "jbuilder"
 gem "jquery-rails"
-gem "kaminari", "< 1.2" # do not upgrade this unless elasticsearch is also upgraded
```

Comment:
[Nice investigation](https://github.com/alphagov/datagovuk_find/pull/1144#issuecomment-1466659512) here. Agree this comment is safe to ignore now 👍 please can you update your commit message so that it's not just captured in a PR comment which can be hard to track down in future?